src/components/common/modal/ModalWrapper.js

```diff
@@ -47,7 +47,7 @@ const ModalWrapper = (props: any) => {
             <ModalInnerContents
               style={[
                 showJaggedEdge ? styles.modalContainerStraightenBottomRightEdge : '',
-                isMobileOnly && styles.nativeBlur,
+                isMobileNative && styles.nativeBlur,
```

Comment:
1. create style for ModalInnerContents in getStylesFromProps
2. use Platform.select to define styles for web, ios/android etc. E.g.
```javascript
Platform.select({
  web: {
    // web styles
    boxShadow: '0px 2px 4px #00000029',
  },
  default: {
    // native styles
    shadowColor: '#000',
    shadowOffset: {
      width: 0,
      height: theme.modals.jaggedEdgeSize,
    },
    shadowOpacity: 0.75,
    shadowRadius: 16,
    elevation: 24,
  }
})
```
3. use that style here

Reply:
Updated with your suggestion, but moved these styles to ModalInnerContents where they are actually used